tools/google-cloud-support-slackbot/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:10
```

Comment:
It's generallty speaking a good practice to use multi-step builds. That is using a base images to compile and build the binary and copy it to a minimal image (alpine or debian-slim). So the final image doesn't contains the build environment which is not needed to run the tool

https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
Fixed!